packages/demos/src/webpack-base-config.js

```diff
@@ -1,8 +1,12 @@
-import {join} from 'path';
-import TsConfigPathsPlugin from 'tsconfig-paths-webpack-plugin';
-import {Configuration} from 'webpack';
+// @ts-check
+const {join} = require('path');
+const TsConfigPathsPlugin = require('tsconfig-paths-webpack-plugin').default;
+const webpack = require('webpack');
 
-export const webpackBaseConfig: Configuration = {
+/**
+ * @type {webpack.Configuration}
+ */
+const webpackBaseConfig = {
```

Comment:
we could also make a direct assignment to `module.exports`

Reply:
Last time I checked, this didn’t work with the type annotation, will check again tomorrow.

Reply:
This is still the case, so unfortunately we need the `const`.